Jenkinsfile

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env groovy
+
+REPOSITORY = 'govuk_taxonomy_helpers'
```

Comment:
It's not in the [A/B testing gem Jenkinsfile](https://github.com/alphagov/govuk_ab_testing/blob/master/Jenkinsfile), and the rails template uses the constant in the integration deployment step. I'm not aware of any Jenkins magic that needs a `REPOSITORY` constant - I think it's safe to just delete.